src/Reporter.cpp

```diff
@@ -422,6 +422,21 @@ namespace geopm
             {"time-hint-unset (s)", {"TIME_HINT_UNSET"}, sample_only},
         };
 
```

Comment:
There is a duplicate include in this file. Please remove the duplicate.
```cpp
#include "geopm_debug.hpp"
```

Reply:
@cmcantalupo this is an existing issue, is there a reason for this or oversight?
https://github.com/geopm/geopm/blob/d964fd09b5c592052a1404949c3777a4b085e025/src/Reporter.cpp#L63